src/main/java/chess/domain/piece/Team.java

```diff
@@ -0,0 +1,15 @@
+package chess.domain.piece;
+
+public enum Team {
+    BLACK,
+    WHITE,
+    EMPTY;
```

Comment:
```suggestion
    WHITE,
    EMPTY,
    ;
```

새로운 enum이 추가될 때를 생각해서 이렇게 쓰기도 한답니다 :)

Reply:
오호... 확실히 새로운 enum을 추가할 때 편리할 것 같아요!!
현재 미션에서는 더 필요한 enum이 추가될 경우가 많지 않을 것 같아서 변경하진 않았지만, 다음에 고려해보겠습니다!! 👍

감사해요! 알렉스